lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -643,6 +645,27 @@ type qemuHostDriveOpts struct {
 	protocol      string
 }
 
+// generateQemuDeviceID generates QEMU device IDs.
+// QEMU device IDs may only be up to 64 characters long, so use a hash if longer.
+func generateQemuDirDriveName(name, prefix, suffix string) string {
```

Comment:
please dont collapse the type of each arg, so define `string` for each arg, thanks